package.json

```diff
@@ -33,38 +34,41 @@
     "test:e2e": "jest e2e/*"
   },
   "dependencies": {
+    "@rollup/plugin-terser": "^0.4.3",
```

Comment:
shouldn't this be a dev dependency?

Reply:
It should, thanks for pointing this out.

Reply:
fixed in #830